### ISSUE TITLE
Add tests for all the components.

### ISSUE THREAD

**Issue by rishichawda** [rishichawda/react-lite-components#27]
- [ ] button
- [ ] header
- [ ] text
- [ ] card
- [ ] image
- [ ] link
- [ ] toggle
- [ ] tooltip

**Comment by rishichawda** [comment]
@joshdenz Yes sure! Thank you for taking the initiative. I've already setup the test framework so you can start working on them!
I'm going to add a few more components in the coming days along with my colleague and then we will also start working on tests and improving coverage. If you have any questions regarding it, feel free to ask them here.

**Comment by joshdenz** [comment]
Sounds great.  I have joined the gitter chat using your invite.
